frontend/pages/DashboardPage/cards/ActivityFeed/ActivityItem/ActivityItem.tsx

```diff
@@ -187,10 +189,22 @@ const TAGGED_TEMPLATES = {
     );
   },
   userChangedGlobalRole: (activity: IActivity, isPremiumTier: boolean) => {
+    const { actor_id } = activity;
+    const { user_id, user_email, role } = activity.details || {};
+
+    if (actor_id === user_id) {
+      // this is the case when SSO user is crated via JIT provisioning
+      // should only be possible for premium tier, but check anyway
+      return (
+        <>
+          was assigned the <b>{role}</b> role{isPremiumTier && " for all teams"}
```

Comment:
Double checking: Does the `&&` trick work here in the false case or does this need to be a ternary to work as expected? 

What renders if you replace `role{isPremiumTier && " for all teams"}` with `role{false && " for all teams"}`?

Reply:
Good instincts – it does seem to work as intended: 
<img width="975" alt="Screenshot 2023-11-20 at 10 26 16 AM" src="https://github.com/fleetdm/fleet/assets/61553566/4748dcce-5817-4b14-8eb8-f16f4a3cd5e3">